<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogpageTest</class>
 <widget class="QMainWindow" name="LogpageTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="2" column="0">
      <widget class="QWidget" name="widget_5" native="true">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px
</string>
        </property>
        <property name="text">
         <string>图像偏移</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>471</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QWidget" name="widget_6" native="true">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px</string>
        </property>
        <property name="text">
         <string>分布图生成</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>471</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="widget_4" native="true">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px</string>
        </property>
        <property name="text">
         <string>图像降噪</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>471</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="widget_3" native="true">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)
</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px
</string>
        </property>
        <property name="text">
         <string>图像矫正</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>471</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">
background-color:rgb(255, 255, 255)</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px</string>
        </property>
        <property name="text">
         <string>图像对齐</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>471</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px</string>
        </property>
        <property name="text">
         <string>图像生成</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>471</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>291</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px</string>
     </property>
     <property name="text">
      <string>恶意逻辑检测</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
